databuilder/query_model/nodes.py

```diff
@@ -276,6 +276,14 @@ class Multiply(Series[Numeric]):
         lhs: Series[Numeric]
         rhs: Series[Numeric]
 
+    class TrueDivide(Series[Numeric]):
+        lhs: Series[Numeric]
+        rhs: Series[Numeric]
+
+    class FloorDivide(Series[Numeric]):
+        lhs: Series[Numeric]
+        rhs: Series[Numeric]
+
```

Comment:
This signature isn't quite right because, as it's written, it's saying that the operation takes any two numeric types and returns a value of _the same numeric type_ as the inputs. But the behaviour we want from `TrueDivide` is that it always returns a float, even if its arguments are ints. And we want `FloorDivide` to always return an int even if its arguments are floats. So the signature should be something like:
```py
class TrueDivide(Series[float]):
    lhs: Series[Numeric]
    rhs: Series[Numeric]

class FloorDivide(Series[int]):
    lhs: Series[Numeric]
    rhs: Series[Numeric]
```